### ISSUE TITLE
does it works with https://www.microsoft.com/store/productId/9N0DX20HK701 ?

### ISSUE THREAD

**Issue by dioda4nic** [wraith13/windows-terminal-vscode#3]
no options added in Windows terminal: open with profile

**Comment by wraith13** [comment]
I haven't updated the link, but the only difference between the link to the preview version and the link to the GA version is the URL, and the actual software is the same.

Please give me more information.

- Is it possible to start Windows Terminal from the Windows Terminal Utility?
- Is it possible to start Windows Terminal with the `wt` command from the command prompt?
- What is the path to Windows Terminal `settings.json`? (It can be displayed from the menu of Windows Terminal.)

**Comment by dioda4nic** [comment]
Yes, i can start a terminal with wt.exe. still no  idea how start with open with profile command. update issue with screenshot  
` "profiles":
    {
        "defaults":
        {
            // Put settings here that you want to apply to all profiles.
        },
        "list":
        [
            {
                // Make changes here to the powershell.exe profile.
                "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
                "name": "Windows PowerShell",
                "commandline": "powershell.exe",
                "hidden": false
            },
            {
                // Make changes here to the cmd.exe profile.
                "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
                "name": "Command Prompt",
                "commandline": "cmd.exe",
                "hidden": false
            },
            {
                "guid": "{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
                "hidden": false,
                "name": "Azure Cloud Shell",
                "source": "Windows.Terminal.Azure"
            }
        ]
    },`

**Comment by wraith13** [comment]
Thank you for the information!

But unfortunately I don't know the cause.

By the way, there was something I didn't understand.
The old Windows Terminal preview( https://www.microsoft.com/p/windows-terminal-preview/9n0dx20hk701 ) published in the store was later released as a GA version ( https://www.microsoft.com/p/windows-terminal/9n0dx20hk701 ) , but a new Windows Terminal preview ( https://www.microsoft.com/p/windows-terminal-preview/9n8g5rfz9xk3 ) was released separately. I'm not sure if this extension works for this new Windows Terminal preview.

**Comment by wraith13** [comment]
I've released v2.1.0, which fixes a bug that might have been related, so check for issues.
